www/c-icap/src/etc/rc.syshook.d/start/50-c-icap

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-# start again to fix race with clamav (no need to restart)
-/usr/local/etc/rc.d/c-icap start
+# restart service to make sure clamav is started
```

Comment:
The comment is wrong now

Reply:
@fichtner Sorry, it was too late yesterday😅